website/content/docs/contributing/index.mdx

```diff
@@ -40,8 +40,8 @@ the call.
 Feel free to join and/or send agenda topics via email, GitHub discussion, or
 on the Matrix server!
 
-[Meeting recordings](https://lf-edge.atlassian.net/wiki/spaces/OP/pages/15211863)
-are available on the LF Edge's Confluence wiki page.
+[Meeting recordings](https://openprofile.dev/my-meetings)
```

Comment:
Oh, I link them in the agenda document: https://docs.google.com/document/d/1HZOjDpm0i2fD-W26FkmMQQStVYFgX3evP-nMpgiGmsA/edit

Reply:
I've also filed the info for the AMER/APAC invite: https://lists.openssf.org/g/openbao/viewevent?repeatid=69620&eventid=2853427&calstart=2025-09-08